modules/javafx.base/src/test/java/test/javafx/collections/ObservableMapTest.java

```diff
@@ -182,6 +180,21 @@ public void testPutAll(Callable<ObservableMap<String, String>> mapFactory) throw
         observer.assertMultipleCalls(call("oFoo", null, "OFoo"), call("pFoo", null, "PFoo"), call("foo", "bar", "foofoo"));
     }
 
+    @ParameterizedTest
+    @MethodSource("createParameters")
+    @SuppressWarnings("unchecked")
+    public void testReplaceAll(Callable<ObservableMap<String, String>> mapFactory) throws Exception {
+        setUp(mapFactory);
+
+        observableMap.replaceAll((key, value) -> switch (key) {
```

Comment:
In addition to testing internal details in ObservableXxxWrapperTests, shouldn't we also test the bulk change functionality inside this test (i.e. testing the publicly accessible Set/Map variants)?

I mean, I would rather prioritize testing of the public APIs than some internal implementation.